monero-outreach-docs/translations/nl/PSA/files/learning.md

```diff
@@ -0,0 +1,18 @@
+# Op zoek naar meer kennis? 
```

Comment:
# Wees steeds leergierig

Ten tijde van dit schrijven wordt het vaak aangenomen dat het merendeel van de Monero-gemeenschap bestaat uit "early adopters", zij die bovengemiddeld technisch zijn. Een paar honderd programmeurs, veel meer dan andere cryptoprojecten, hebben aan Monero's code gewerkt. De achterliggende technologie is moeilijker te begrijpen voor buitenstaanders, en misschien is het hierdoor dat de koerskloof tussen Bitcoin en Monero nog niet gedicht is. Maar wij, early adopters, blijven voorwaarts gaan in de hoop Monero erkenning en aanvaarding te geven. Na 100 jaar zal Monero nog altijd van mensen afhangen, mensen zoals jou, die het ecosysteem bewaken en de code nog meer verbeteren. Dit zijn de redenen waarom we aanraden dat iedereen zijn kennis verdiept, vooral als je geen programmeur bent. Vergroot je "Monero-geletterdheid" en consumeer zoveel mogelijk informatie als je kan.

Door meer te leren over de functies, code, het onderliggend open-bron- en cypherpunkethos, en over hoe beveiliging en privacy te verbeteren tijdens gebruik, kunnen we samen Monero's ecosysteem versterken. Diep inzicht in alle aspecten van Monero helpt anderen erover aan te spreken en adoptie te versnellen. Je zal je comfortabeler en zekerder voelen in het bewaren, verzenden en ontvangen van XMR. Kennis is macht, en kan de wereld veranderen. Wees steeds leergierig.. en doe je eigen research!

### De toekomst begint bij jezelf, hier is wat leesmateriaal om meer te leren over Monero:

**_[getmonero.org (Official Website)](https://getmonero.org/)_**
**_[monero.how](https://www.monero.how/)_**
**_[reddit.com/r/monero](https://www.reddit.com/r/Monero/)_**
**_[Guide to Monero (post)](https://www.reddit.com/r/CryptoCurrency/comments/7ra409/your_guide_to_monero_and_why_it_has_great/)_**
**_[Monero In-Depth Technical Intro](https://steemit.com/monero/@sgp/7yjqso-a-monero-introduction-for-beginners)_**
**_[Zero to Monero](https://www.getmonero.org/library/Zero-to-Monero-1-0-0.pdf)_**
**_[Scams to Avoid](https://www.reddit.com/r/Monero/wiki/avoid)_**
**_[Monero FAQ](https://ww.getmonero.org/get-started/faq/)_**
**_[Connect w/ Monero Community](https://getmonero.org/community/hangouts/)_**
**_[Mastering Monero - (coming soon)](https://masteringmonero.com/)_**